client/src/pages/Plantkins.js

```diff
@@ -61,7 +88,10 @@ function Plants() {
         setUser(res.data[0]);
       })
       .catch((err) => console.log(err));
-  }, [user]);
+    
```

Comment:
On line 64, please add user back in.  This is what actually forces a refresh of the page after the favorite button is clicked.